src/cmds/scripts/pbs_habitat.in

```diff
@@ -655,10 +672,12 @@ if [ "${PBS_START_SERVER:-0}" != 0 ] ; then
 			exit $ret
 		else
 			#Remove old dataservice directory
-			if [ `echo "${old_inst_dir}"| grep "pgsql.old"` ]; then
-				err=`rm -rf "${old_inst_dir}"`
+			if [ -d "${old_inst_dir}" ]; then
+				backupdir "$old_inst_dir" "$PBS_HOME"
```

Comment:
This is an existing code, was present in spec file before this change.
I guess this has been kept for the user, in case they want to downgrade to old pgsql.
I forgot to remove the comment.

Reply:
removed the comment